composer.json

```diff
@@ -35,5 +35,9 @@
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         }
-    }
+    },
+    "bin": [
+        "bin/doctrine-migrations",
+        "bin/doctrine-migrations.php"
```

Comment:
And this is a mistake in the ORM (it was reverted in DBAL but I missed the fact that the ORM included it too)